package.json

```diff
@@ -2,10 +2,11 @@
   "name": "phonegap-docs",
   "version": "0.0.1",
   "description": "The official PhoneGap documentation.",
+  "repository": "git@github.com:phonegap/phonegap-docs.git",
   "dependencies": {
     "autoprefixer-stylus": "^0.5.0",
-    "docpad": "6.73.5",
-    "docpad-plugin-browsersync": "./vendor/docpad-plugin-browsersync",
+    "docpad": "^6.78.1",
+    "docpad-plugin-browsersync": "^2.0.6",
```

Comment:
Should we not be vendoring this plugin instead of using the published version that is not compatible with node `0.12.x`?